### DISCUSSION TITLE
Upgrade Installation scheduling

### DISCUSSION THREAD

**Discussion by bmcnally-ghx** [umbraco/Umbraco.Cloud.Issues#794] [Ideas]
### Issue description

We are located in North American and quite often upgrades/updates are scheduled during our prime site visitation hours and often require a restart. How can we have more control over this timing so that our site does not become unavailable during it's highest traffic windows?

Thanks
-Brian

**Comment by sajumb** [comment]
Unfortunately, we currently do not offer scheduled auto upgrades.

We have plans to support this and currently it is located under "Later" in our public roadmap (https://umbraco.com/products/roadmap/).

The auto-upgrades of CMS, Forms, and Deploy start Tuesdays at 7 AM Central European Time (CET) but the updating of all products and all projects spans most of the day.
If you have a great need to be in control of the time, then it can be an option to exclude a project from auto-updates. If so, then simply reach out to our support and they will be able to help with this.
